5calls/app/src/main/java/org/a5calls/android/a5calls/adapter/IssuesAdapter.java

```diff
@@ -472,4 +490,33 @@ public EmptySearchViewHolder(View itemView) {
     }
 }
 
+    /**
+     * Sorts a list of issues to prioritize those with meta values (state abbreviations) at the top,
+     * then sorts the remaining issues. Both groups maintain their internal sort order.
+     */
+    @VisibleForTesting
+    ArrayList<Issue> sortIssuesWithMetaPriority(List<Issue> issues) {
```

Comment:
In a local build of this, I'm getting a non-state issue above a state issue, seems incorrect?

<img width="1034" height="1838" alt="Image" src="https://github.com/user-attachments/assets/baeb9028-233c-42a1-bf32-c440e2d74c3e" />

Reply:
data issue, fixed this one in the db